engineering/ember.md

```diff
@@ -116,8 +135,35 @@ fullName: computed('firstName', 'lastName', {
 
 ```
 
+## Glimmer Components
+
+### Component Args
+
+- constructor with setting local props, args with objects, chain in template or getter in JS?
+
+### Tracked properties
+
+- Concerns for mixed use of computed props passed down from existing controllers
+
+### Computed properties
+
+- Using computed properties instead of getter and setter
+
+
+## Tailwind CSS
+
+- Using important when tailwind css is overwritten or adding to scss?
+
+
+
 ## Templates
 
+### Actions
+
+- Triggering single boolean property, use `action` or `mut`?
```

Comment:
`mut` has old backwards compatibility for v1 days, I think. it sets and reads. 

docs:
- https://api.emberjs.com/ember/3.25/classes/Ember.Templates.helpers/methods/mut?anchor=mut

articles:
- https://www.pzuraq.com/on-mut-and-2-way-binding/
- https://v5.chriskrycho.com/journal/mut-and-set-and-auto-tracking-in-ember-octane/
- https://gist.github.com/Joelkang/dd7f3f54dc2f1115526dbca001853dbe

Reply:
Side note: I find `(fn (mut this.booleanishProperty) true|false)` a better alternative to actions that toggle a property. Especially if its a property on a controller which lives on after you leave its route. I've seen issues where showModal is still true when you revisit the route via back button etc.